### ISSUE TITLE
libclang dependency no longer available in unstable 

### ISSUE THREAD

**Issue by barnumbirr** [barnumbirr/delta-debian#2]
It seems the `libclang-6.0-dev` that was previously available in all three Debian versions used to build the `delta` package [is no longer available for in `unstable`](https://tracker.debian.org/news/1148022/removed-1601-141-from-unstable/). This was uncovered when building the new package version after fixing #1 .

```bash
2020-06-08T22:58:24.3124511Z Selecting previously unselected package delta-diff-build-deps.
2020-06-08T22:58:24.3214750Z (Reading database ... 29250 files and directories currently installed.)
2020-06-08T22:58:24.3225888Z Preparing to unpack delta-diff-build-deps_0.1.1-2_all.deb ...
2020-06-08T22:58:24.3241172Z Unpacking delta-diff-build-deps (0.1.1-2) ...
2020-06-08T22:58:24.9495240Z Reading package lists...
2020-06-08T22:58:25.1056814Z Building dependency tree...
2020-06-08T22:58:25.1061642Z Reading state information...
2020-06-08T22:58:25.1329699Z Starting pkgProblemResolver with broken count: 1
2020-06-08T22:58:25.1562486Z Starting 2 pkgProblemResolver with broken count: 1
2020-06-08T22:58:25.1587726Z Investigating (0) delta-diff-build-deps:amd64 < 0.1.1-2 @iU mK Nb Ib >
2020-06-08T22:58:25.1588164Z Broken delta-diff-build-deps:amd64 Depends on libclang-6.0-dev:amd64 < none @un H >
2020-06-08T22:58:25.1588498Z   Removing delta-diff-build-deps:amd64 because I can't find libclang-6.0-dev:amd64
2020-06-08T22:58:25.1608729Z Done
```

`libclang-8-dev` could be a suitable replacement but is only available in the  `buster-backports` repository.

**Comment by barnumbirr** [comment]
We could also not enforce a version at all and use `libclang-dev` as a dependency.

**Comment by barnumbirr** [comment]
Resolved in [v0.1.1-2](https://github.com/barnumbirr/delta-debian/releases/tag/v0.1.1-2)
